src/data/adventures.txt

```diff
@@ -705,3 +705,23 @@ PirateRealm Island	adventure=531	DiffLevel: mid Env: outdoor Stat: 0 nowander	St
 PirateRealm Island	adventure=531	DiffLevel: mid Env: outdoor Stat: 0 nowander	Red Roger's Fortress
 PirateRealm Island	adventure=531	DiffLevel: mid Env: outdoor Stat: 0 nowander	Jack's Hideout
 PirateRealm Island	adventure=531	DiffLevel: mid Env: outdoor Stat: 0 nowander	The Temple
+
+# Obselete areas that nevertheless are reported as possibilities for the autumnaton
+Plains	adventure=18	DiffLevel: unknown Env: unknown Stat: 10	Degrassi Knoll OBE
```

Comment:
OBE? "Order of the British Empire"? "Out of Body Experience"?

Make them all (OLD)

What happens if you go to adventure.php?snarfblat=18?
I assume it fails.

I feel strongly that unless you can actually adventure there, they do not belong in "Plains" or "Manor2" or whatever.

canAdventure purports to handle EVERY adventure that is in a particular zone.
These do not belong in those switch statements.

Make a new zone in zonelists.txt - Obsolete - with parent Removed.
Make all these locations have zone Obsolete

Reply:
OK on Zones.  Will work on later.